storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc

```diff
@@ -0,0 +1,34 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="persistent-storage-csi-smb-cifs"]
+= SMB CIFS CSI Driver Operator
+include::_attributes/common-attributes.adoc[]
+:context: persistent-storage-csi-smb-cifs
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Server Message Block (SMB) protocol Common Internet File System (CIFS) dialect.
+
+:FeatureName: SMB CIFS CSI Driver Operator
```

Comment:
```suggestion
:FeatureName: CIFS/SMB CSI Driver Operator
```
Maybe better to keep the same name with the display name https://github.com/openshift/csi-operator/blob/79ca0ac2df11f8e047e3eae4e08f13bf745814f1/Dockerfile.samba#L10 .